gym/vector/sync_vector_env.py

```diff
@@ -161,7 +187,13 @@ def call(self, name, *args, **kwargs):
 
         return tuple(results)
 
-    def set_attr(self, name, values):
+    def set_attr(self, name: str, values: Union[list, tuple, Any]):
+        """Sets the attribute of a property with values.
+
+        Args:
+            name: The property name to change
+            values: The value to set the property
```

Comment:
Use same text as in async_vector_env
```suggestion
            values: Values of the property to be set to. If ``values`` is a list or
                tuple, then it corresponds to the values for each individual
                environment, otherwise, a single value is set for all environments.
```